wazuh/wazuh-kibana-app
# Fix deploy new agent section

### Description

- Fixed the way macOS versions and architectures were displayed.
- Fixed the way agents were displayed.
- Fixed the way Ubuntu versions were displayed.
- Added extra steps message and new command for Windows XP and Windows Server 2008
- Added Alpine agent with all its steps.
- Windows command fixed
- CI/CD has [updated](https://github.com/wazuh/wazuh-packages/issues/1625#issuecomment-1357808349) all installation commands, we have also updated them in our code.
 
### Issues Resolved

#4931 

### Manual Backports

[4.4-2.3-wzd](https://github.com/wazuh/wazuh-kibana-app/pull/4974)
[4.4-7.16 ](https://github.com/wazuh/wazuh-kibana-app/pull/4979)

### Evidence

https://user-images.githubusercontent.com/99441266/204812848-a0be5f6d-15c3-4a2e-b893-a4b0f0016ec4.mov

### Steps to test

* Click on agents
* Click on deploy new agent
* Check that the information shown is correct.
* Check that the information shown for the windows agent is the one shown in this [comment](https://github.com/wazuh/wazuh-packages/issues/1625#issuecomment-1320226193)
* Check that the information shown for the alpine agent is the same as the one shown in this [comment](https://github.com/wazuh/wazuh-packages/issues/1625#issuecomment-1328980215)

### Check List
